rpc/client/live.go

```diff
@@ -14,7 +14,7 @@ import (
 func Register(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	n := &node.Node{}
 	if err := shared.PopModel(w, r, ps, n); err != nil {
-		shared.WriteJSONResponse(w, "500 error: "+err.Error())
+		shared.WriteErrorResponse(w, 500, err.Error())
```

Comment:
I think this is more likely a 400 Bad Request

Reply:
Yes agreed, a badly formatted request should be the only reason for error. Will fix this